### DISCUSSION TITLE
The Black Death timeline?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#762] [Galaxy Trivia] [answered]
.

**Comment by peterbaccaro** [comment, marked as answer]
The Black Death was a devastating pandemic that swept through Europe in the mid-14th century, killing an estimated 25 million people, or about one-third of the population. Here is a timeline of some of the major events associated with the Black Death:

- **1347**: The Black Death arrives in Europe from the East. It is believed to have originated in the region around China and spread along trade routes, reaching Europe via the port of Genoa in Italy.
- **1348**: The Black Death spreads rapidly throughout Europe, with outbreaks occurring in France, Spain, Germany, England, and other countries. Many cities and towns are hit particularly hard, with death tolls often reaching 50% or higher.
- **1349**: The Black Death reaches its peak in Europe, with the worst outbreaks occurring in the summer months. By this time, many people have become aware of the seriousness of the disease and are taking precautions such as quarantining the sick and avoiding public gatherings.
- **1350**: The number of deaths from the Black Death begins to decline, although sporadic outbreaks continue to occur for several years. It is estimated that by the time the pandemic has run its course, as many as 75 million people may have died worldwide.
- **1361**: A second major outbreak of the Black Death occurs, centered in India and spreading throughout Asia and the Middle East. This outbreak is less severe than the one that occurred in the 14th century, but it still causes significant loss of life.
- **1665**: A smaller outbreak of the Black Death occurs in England, centered in London. This outbreak is believed to have been caused by a different strain of the bacteria responsible for the original pandemic. It causes several thousand deaths before being brought under control.

The Black Death was one of the deadliest pandemics in human history and had a profound impact on European society and culture. Its effects were felt for centuries, leading to changes in the way people lived, worked, and thought about illness and death.
